packages/flutter_tools/lib/src/build_runner/build_script.dart

```diff
@@ -289,12 +288,7 @@ Future<void> main() async {
 }
 
 void internalBootstrapBrowserTest(Function getMain()) {
-  var channel =
-      serializeSuite(getMain, hidePrints: false, beforeLoad: () async {
-    var serialized =
-        await suiteChannel("test.browser.mapper").stream.first as Map;
```

Comment:
I'm _pretty_ sure this is safe to do today and will let us remove the class in an upcoming change. One thing I'm a little unsure of is whether not removing this `beforeLoad` and waiting for _something_ on this channel will change timing in a negative way. I think the value was always `null` since we never passed anything to the `mapper` named argument to `load()`

cc @grouma - does anything stand out to you here and unsafe?

cc @jonahwilliams - what is the best way to validate that there is no regression?

Reply:
Discussed offline. This should be safe to do assuming that a null `beforeLoad` does not impact the timing of tests in some negative way.